.github/workflows/update-css.yml

```
@@ -0,0 +1,89 @@
+  release:
+    types: [published]
+    repositories:
+      - sindresorhus/github-markdown-css
+
+jobs:
+  update-css:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)